joomla/joomla-cms
# [4.0] Add the new doc page to the HTTP Headers Plugin postinstall message

### Summary of Changes

- Add the new doc page to the HTTP Headers Plugin postinstall message
- add other Explanation links.

### Testing Instructions

- Install 4.0
- disable the plg_system_httpheaders Plugin
- check the postinstall

### Expected result

![image](https://user-images.githubusercontent.com/2596554/56849487-5bec4d80-68f5-11e9-8dbe-d20d3eade711.png)


### Actual result

![image](https://user-images.githubusercontent.com/2596554/56501195-25d25680-650e-11e9-97e1-0331d59d121d.png)


### Documentation Changes Required

none
